code/core/src/manager/settings/shortcuts.tsx

```diff
@@ -57,6 +57,7 @@ export const Description = styled.div({
 
 export type ValidationStates = 'valid' | 'error' | 'warn';
 
+// @ts-expect-error (non strict)
```

Comment:
TypeScript tells me this comment is not necessary

Reply:
This error only shows in prod mode:
```
src/manager/settings/shortcuts.tsx:60:14 - error TS2322: Type 'StyledComponent<Omit<Omit<HTMLProps<HTMLInputElement>, "align" | "height" | "size" | "valid"> & { size?: Sizes | undefined; align?: Alignments | undefined; valid?: ValidationStates | undefined; height?: number | undefined; }, "ref"> & ... 5 more ... & { ...; }, {}, {}>' is not assignable to type 'FC<Omit<Omit<HTMLProps<HTMLInputElement>, "align" | "height" | "size" | "valid"> & { size?: Sizes | undefined; align?: Alignments | undefined; valid?: ValidationStates | undefined; height?: number | undefined; }, "ref"> & ... 4 more ... & { ...; }>'.
  Types of parameters 'props' and 'props' are incompatible.
    Type 'Omit<Omit<HTMLProps<HTMLInputElement>, "align" | "height" | "size" | "valid"> & { size?: Sizes | undefined; align?: Alignments | undefined; valid?: ValidationStates | undefined; height?: number | undefined; }, "ref"> & ... 4 more ... & { ...; }' is not assignable to type 'Omit<Omit<HTMLProps<HTMLInputElement>, "align" | "height" | "size" | "valid"> & { size?: Sizes | undefined; align?: Alignments | undefined; valid?: ValidationStates | undefined; height?: number | undefined; }, "ref"> & ... 5 more ... & { ...; }'.
      Type 'Omit<Omit<HTMLProps<HTMLInputElement>, "align" | "height" | "size" | "valid"> & { size?: Sizes | undefined; align?: Alignments | undefined; valid?: ValidationStates | undefined; height?: number | undefined; }, "ref"> & ... 4 more ... & { ...; }' is not assignable to type '{ theme?: Theme | undefined; }'.
        Types of property 'theme' are incompatible.
          Type 'Theme | undefined' is not assignable to type 'import("/Users/me/Projects/Storybook/mono/storybook/code/core/dist/theming/index").Theme | undefined'.
            Type 'Theme' is missing the following properties from type 'Theme': color, background, typography, animation, and 11 more.

60 export const TextInput: FC<ComponentProps<typeof Form.Input> & { valid: ValidationStates }> =
                ~~~~~~~~~
```